server-side/controllers/visitedWebSite.controller.js

```diff
@@ -1,53 +1,52 @@
-import visitedWebsites from '../models/visitedWebSite.model.js'; 
+import VisitedWebsite from '../models/visitedWebSite.model.js';
 
- export const    getAllVisitedWebsites= async (req, res) => {
-        try {
-            const visitedWebsites = await visitedWebsites.find();
-            res.json(visitedWebsites);
-        } catch (err) {
-            res.status(500).json({ message: err.message });
+export const getAllVisitedWebsites = async (req, res,next) => {
+    try {
+        const visitedWebsites = await VisitedWebsite.find().populate('websiteId').select('-__v');
+        res.json(visitedWebsites);
+    } catch (err) {
+        next({message:err.message})
```

Comment:
always send with suitable status code
also add return after the call to next
fix all occurrences in file